src/client/pages/OfferDebugger/components/QuoteFormSweden.tsx

```diff
@@ -3,6 +3,24 @@ import { ApartmentType } from 'data/graphql'
 import { InputField } from 'components/inputs'
 import { WithFormikProps } from './QuoteData'
 
+// Not sure what the data here should be, but this makes it work
+export const initialSeHouseValues = {
+  firstName: 'Sven',
+  lastName: 'Svensson',
+  currentInsurer: '',
+  birthDate: '1995-09-29',
+  ssn: '199509291234',
+  startDate: '',
+  email: 'sven.svensson@hedvig.com',
+  data: {
+    street: 'Storgatan 1',
+    zipCode: '12345',
+    livingSpace: 23,
+    numberCoInsured: 1,
+    subType: ApartmentType.Rent,
+  },
+}
+
```

Comment:
Nice that you added this!

Some things are missing that should be required:

- ancillaryArea
- yearOfConstruction
- numberOfBathrooms

`subType` is not needed for house so maybe remove that

Reply:
I think it's fine to leave them out -- you can always change that in the edit modal

Reply:
Ok, removed `subType` from house. The other fields you mentioned are set hard-coded when submitting the form 🤷